game/views.py

```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
-
+from django.http import HttpResponse
+from .models import Team
 # Create your views here.
+def index(request):
```

Comment:
index to zbyt ogólna nazwa. Można to przemianować na scoreboard bo tym właśnie ta strona jest.